src/main/java/com/fasterxml/jackson/databind/AnnotationIntrospector.java

```diff
@@ -1149,6 +1150,14 @@ public String[] findEnumValues(Class<?> enumType, Enum<?>[] enumValues, String[]
         return names;
     }
 
+    /**
+     * @since 2.15
+     */
+    public String[] findEnumValues(MapperConfig<?> config, Class<?> enumType, Enum<?>[] enumValues,
```

Comment:
We can (and I think should) combine `enumType` and `annotatedClass`; just make `enumType` of type `AnnotatedClass`. `Class<?>` can be gotten using `AnnotatedClass.getRawType()`. This way signature change is bit smaller.